repo/packages/G/gestalt-framework/1/resource.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images": {
+    "icon-large": "https://s3.amazonaws.com/gfi.misc/Gestalt-Icon-Large.png",
+    "icon-medium": "https://s3.amazonaws.com/gfi.misc/Gestalt-Icon-Medium.png",
+    "icon-small": "https://s3.amazonaws.com/gfi.misc/Gestalt-Icon-Small.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
```

Comment:
We'd like to have a new version for each release (even hotfixes). We're going to have a more automated process here soon to make it way easier to get new versions in and going.